tests/test_model_serializer.py

```diff
@@ -381,6 +382,10 @@ class Meta:
             TestSerializer():
                 id = IntegerField(label='ID', read_only=True)
                 duration_field = DurationField(max_value=datetime.timedelta(3), min_value=datetime.timedelta(1))
+        """) if sys.version_info < (3, 7) else dedent("""
```

Comment:
What's happening here? (I need a ☕️)

Reply:
Python 3.7 changed the `timedelta` repr to now include the argument names. So, previously this was `timedelta(3)` and now it's `timedelta(days=3)`. This check is necessary for compatibility. IDK if there is a better way of testing the python version here.